loader-img-full/checkBdev.sh

```diff
@@ -187,6 +187,7 @@ if [ "$android" != "true" ] && [ "$batoceraSquashfs" != "true" ]; then
             otherDistroColorLen="12 10"
             ;;
         artix)
+	    	isArtix=true #for the init check system
```

Comment:
Fix the indentation, add a space after the `#` in the comment, and fix the grammar of the comment